integrationtests/updateNFT_test.go

```diff
@@ -36,18 +36,19 @@ func TestNFTUpdateMetadata(t *testing.T) {
 	header := &dataBlock.Header{
 		Round:     50,
 		TimeStamp: 5040,
-		ShardID:   2,
+		ShardID:   1,
```

Comment:
Same remark for shardID

Reply:
Changed the shard ID because the new address  used in the test is in shard 1